### PR TITLE
Polish translation (not 100% accurate).

### PR DESCRIPTION
**Pull request by Behoston**
It's not possible to translate properly `grand` to Polish. In English you say `grand+mother` or `grand+father` in PL we have separate words `babcia` and `dziadek`. In game `fathers` doesn't exists for now, but still I can't find good way to translate grand. Maybe something like mother of mother.

Second problem: in PL we have different suffix for word `your` that depends on sex. For female is `twoja` for male is `twój`. I used `twój/twoja` but `/` doesn't appear in game and even if displayed, is not good way.

For some strings I don't know context.

Many strings are missing (info messages, tool names etc).

I'd prefer to not merge this branch for now.

### CODE REVIEW

**Review by KubsHD — CHANGES_REQUESTED** [review]
Text on line 248 should say "WCIŚNIJ" instead od "WSIŚNIJ"

File: gameSource/languages/Polish.txt [hunk 1]
```diff
@@ -245,7 +245,7 @@ pauseMessage1 "NA PEWNO WYJŚĆ?"
 pauseMessage2 "WCIŚNIJ [ENTER] ABY KONTYNUOWAĆ GRĘ"
 
 
-pauseMessage3 "WSIŚNIJ [%] ABY WYJŚĆ"
+pauseMessage3 "WCIŚNIJ [%] ABY WYJŚĆ"
 pauseMessage3b "NIE MOŻESZ TEARZ WYJŚĆ"
```
Comment (KKSzymanowski) [inline gameSource/languages/Polish.txt]: **TEARZ** -> **TERAZ**

File: gameSource/languages/Polish.txt [hunk 2]
```diff
@@ -0,0 +1,276 @@
+writeFailed "BŁĄD:  FOLDER Z GRĄ JEST W TRYBIE TYLKO OD ODCZYTU"
+
+
+measuringFPS "MIERZENIE MAKSYMALNYCH FPS"
+
+vsyncWarning "OSTRZERZENIE: SYNCHRONIZACJA PIONOWA WYŁĄCZONA"
```
Comment (KKSzymanowski) [inline gameSource/languages/Polish.txt]: **OSTRZERZENIE** -> **OSTRZEŻENIE**

File: gameSource/languages/Polish.txt [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+transitions "TŁUMACZENIA"
+animations "ANIMACJE"
+categories "KATEGORIE"
+
+groundTextures "TEKSTURY PODŁOŻA"
+
+
+spritesRebuild "GRAFIKI##(PRZEBUDOWYWANIE CACHE)"
+objectsRebuild "OBIEKTY##(PRZEBUDOWYWANIE CACHE)"
+transitionsRebuild "TŁU,ACZENIA##(PRZEBUDOWYWANIE CACHE)"
```
Comment (KKSzymanowski) [inline gameSource/languages/Polish.txt]: Shouldn't it be **TŁUMACZENIA**?

File: gameSource/languages/Polish.txt [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+
+
+backButton "POWRÓT"
+restartButton "RESTART"
+fullscreen "PEŁNY EKRAN:"
+musicLoudness "GŁOŚNOŚĆ MUZYKI:"
+soundLoudness "GŁOŚNOŚĆ DŹWIĘKÓW:"
+
+currentFPS "AKTUALNA ILOŚĆ KLATEK:"
+targetFPS "DOCELOWA ILOŚĆ KLATEK:"
```
Comment (KKSzymanowski) [inline gameSource/languages/Polish.txt]: I'd go with "liczba klatek" instead of "ilość klatek".

File: gameSource/languages/Polish.txt [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+vsyncOn "SYNCHRONIZACJA PIONOWA:"
+vsyncYes "TAK"
+vsyncNo "NIE"
+redetectButton "WYKRYJ PONOWNIE"
+
+
+downloadingUpdate "POBIERANIE AKTUALIZACJI"
+applyingUpdate "INSTALOWANIE AKTUALIZACJI"
+
+serverShutdownMessage "SERWERY ZOSTANĄ NIEBAWEM##WYŁĄCZONE Z PRZYCZYN TECHINCZNYCH.##PPROSZĘ SPRÓBOWAĆ PÓŹNIEJ."
```
Comment (KKSzymanowski) [inline gameSource/languages/Polish.txt]: **PPROSZĘ** -> **PROSZĘ**

**Review by KKSzymanowski — CHANGES_REQUESTED** [review]

File: gameSource/languages/Polish.txt (hunk 4 above)
Comment (Behoston) [reply gameSource/languages/Polish.txt]: You are right:
https://www.youtube.com/watch?v=Gc31UQ-C6dw

**Comment by Behoston** [comment]
I've changed color scheme in my IDE :P
All misspelled words fixed.

**Comment by jasonrohrer** [comment]
Sorry, I don't maintain translations in my code base.  There are simply too many languages in the world, and I'm just one developer.

Please coordinate with people in the forums to distribute your translation.

**Pull request closed without merging**